packs/clicrud/config.yaml

```diff
@@ -0,0 +1,5 @@
+---
+  username: "admin"
```

Comment:
I pushed a change to master to replace config.yaml with config schema - 54d5534e3d437aed79f903b6327cd42103afa935